### PR TITLE
0.9.0 -> 0.9.0-rc1

### PR DESCRIPTION
**Pull request by leoisl**
Change 0.9.0 -> 0.9.0-rc1.

Also has a small fix to allow us to put labels on versions (`CMake` does not allow labels on versions, just major, minor and patch versioning, and these have to be numbers).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -130,3 +130,5 @@ example/venv
 /example/pandora-linux-precompiled-v0.9.0.gz
 
 example/pandora-linux-precompiled-pandora_paper_tag1
+
+example/pandora-linux-precompiled-v0.9.0-rc1
```
Comment (mbhall88) [inline .gitignore:134]:
```suggestion
example/pandora-linux-precompiled*
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ project adheres to
 
 ## [Unreleased]
 
-## [0.9.0]
+## [0.9.0-rc1]
```
Comment (mbhall88) [inline CHANGELOG.md:12]: When you wrap this in square brackets and don't add a link after you need to make sure there is a defined link defined elsewhere. i.e. check near the bottom for examples, create a link `[0.9.0-rc1]: <URL>`

**Review by mbhall88 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (leoisl) [reply .gitignore]: I actually ended up cleaning this `.gitignore` file as there were so many outdated entries. Could you please re-review?

File: CHANGELOG.md (hunk 2 above)
Comment (leoisl) [reply CHANGELOG.md:12]: Yep, sorry for missing that! Fixed this in the `CHANGELOG.md` and in the pre-release description. Also fixed a small issue with the `Unreleased` link, could you please re-review?

**Review by mbhall88 — APPROVED** [review]

**Pull request merged**